L/libjwt/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libjwt"
+version = v"1.15.3"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/benmcollins/libjwt.git", "e8af37ce2e7de2d3bbadaaf232dc6f6b0fd97f03")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/libjwt
+autoreconf -i
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("aarch64", "macos"; ),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("x86_64", "macos"; ),
+    Platform("aarch64", "linux"; libc = "glibc")
+]
```

Comment:
Any reason for not supporting more platforms?

Reply:
This was what I needed for my purposes, I will increase coverage now!